src/mower_logic/src/mower_logic/mower_logic.cpp

```diff
@@ -483,6 +484,14 @@ void reconfigureCB(mower_logic::MowerLogicConfig &c, uint32_t level) {
     last_config = c;
 }
 
+bool startInAreaCommand(mower_msgs::StartInAreaSrvRequest &req, mower_msgs::StartInAreaSrvResponse &res) {
```

Comment:
I think this should this really be an action in the behaviours. Issue is passing params to an action, @ClemensElflein any ideas for how this could be done?

Reply:
yes, that would be nice, because this way we could call it via MQTT. The issue is, that actions currently don't have parameters.

Reply:
So this can only be called using a ROS service call?

I think we can merge it for now due to lack of a better option. Also we have an implementation we can then adapt as soon as the xbot actions can accept parameters.